MeetupPlatform.Api/Features/Meetups/Contacts/AddContact/AdditionContactDto.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace MeetupPlatform.Api.Features.Meetups.Contacts.AddContact;
+
+using System.ComponentModel.DataAnnotations;
+
+public class AdditionContactDto
+{
+    /// <summary>
+    /// Title of contact.
+    /// </summary>
+    /// <example>Phone number.</example>
+    [Required]
+    [MaxLength(50)]
+    public string Title { get; set; }
+
+    /// <summary>
+    /// Value of contact.
+    /// </summary>
+    /// <example>80(33)-374-21-24</example>
+    [Required]
+    [MaxLength(1000)]
+    public string Value { get; set; }
```

Comment:
Perhaps some validation is required here so that when writing a phone number, the user cannot write "WADQWDSDQWDDASDQWDQWDQWD".

Reply:
This might also be a URL or some other kind of contact, so simple alphanumeric filter is not suitable here

Reply:
so I suggest doing it this way, giving the user a choice of what he will add at the moment, email, phone, or something else, and for each type of contact create his own model and use, for example, EmailAddressAttribute() for validation

Reply:
We won't benefit from this right now, so I would recommend against doing so